### ISSUE TITLE
invalid parenthesis code

### ISSUE THREAD

**Issue by erickAvSanti** [naitoh/rbpdf#48]
I am writting html "(Global positioning System)".. so rbpdf write in utf8 mode "....a GPS #!#OP#!#Global Positioning System#!#CP#!#, homolo", how is possible ?

<div style="text-align:justify;width:100%;font-size:9;">
	 Tecnología GPS (Global Positioning System)
</div>

**Comment by naitoh** [comment]
I can not understand what you want to do, but you can write like the one below.

* app/controllers/exampleXXX_controller.rb
```
class Example000Controller < ApplicationController
  def index
    # create new PDF document
    pdf = RBPDF.new
    
    # set font
    pdf.set_font('times', 'BI', 20)
    
    # add a page
    pdf.add_page()
    
    pdf.write(0, "(Global positioning System)")

    html = "(Global positioning System)"
    pdf.write_html_cell(0, 0, '', '', html, 0, 1, 0, true, '', true)

    html = "....a GPS #!#OP#!#Global Positioning System#!#CP#!#, homolo"
    pdf.write_html_cell(0, 0, '', '', html, 0, 1, 0, true, '', true)

    # Close and output PDF document
    send_data pdf.output(), :type => "application/pdf", :disposition => "inline"
  end
end
```

![rbpdf_global_positioning_system](https://user-images.githubusercontent.com/706220/51914289-db17d580-241b-11e9-9cf1-e5102a9e0cb2.png)

FYI
https://github.com/naitoh/rbpdf/wiki/Examples

**Comment by erickAvSanti** [comment]
try this code.


html = "<div style=\"text-align:justify;\">hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) hello Ruby (inside hello world) </div>"; 
	    pdf.write_html(html,true,0,true,true)

**Comment by erickAvSanti** [comment]
![image](https://user-images.githubusercontent.com/12147984/51926726-e806db80-23bf-11e9-8c5f-9188f38e2628.png)

**Comment by erickAvSanti** [comment]
reading your code, it behave when text align is justify "J"

**Comment by naitoh** [comment]
Thank you for your report.
This problem is a bug in rbpdf. a31efe9e3a025b9b538ef7e11bb096a612246d66

After correction, it is displayed as follows.
![rbpdf_fix](https://user-images.githubusercontent.com/706220/51985442-2e079080-24e1-11e9-8eb0-e2aac098020f.png)

**Comment by naitoh** [comment]
This problem fixed by rbpdf 1.19.8.
Thanks!
